charts/templates/clustersynchro-manager-deployment.yaml

```diff
@@ -37,7 +37,6 @@ spec:
         - /usr/local/bin/clustersynchro-manager
         - --storage-config=/etc/clusterpedia/storage/internalstorage-config.yaml
         - --leader-elect-resource-namespace={{ .Release.Namespace }}
-        - --feature-gates=PruneManagedFields=true,PruneLastAppliedConfiguration=true
```

Comment:
Would it be better to set it explicitly for the feature gate we define and use?

Reply:
The feature gate is beta at current stage, we can set it explicitly.  if the feature gate update to GA, we don't have to specify it, do you think?

Reply:
> The feature gate is beta at current stage, we can set it explicitly. if the feature gate update to GA, we don't have to specify it

I think it's fine